### DISCUSSION TITLE
Text Alignment options on Text Box Visualization not working

### DISCUSSION THREAD

**Discussion by tinoptilogic** [RevealBi/Reveal.Sdk#119] [General]
The text alignment for Text Boxes used in dashboards doesn't seem to be working. I have a test model which has a single dashboard that should be displaying the text box with text alignment to the right. The text will stay left-aligned no matter what is selected in the option.

![image](https://github.com/RevealBi/Reveal.Sdk/assets/150904348/8d4f77f7-b235-465d-a96a-81c6be914483)

**Comment by TorreyBetts** [comment]
What browser are you using and SDK version? In the future when creating these, add some context about the OS, SDK type (WPF, NodeJS, Java, etc), and the version used.

**Reply by tinoptilogic** [reply]
SDK version is 1.2.3 and browser is chrome. In the future will try to give more information for tickets. Thanks

**Reply by TorreyBetts** [reply]
I'm nearly 100% positive that this was fixed some time this year. Once you update, this issue should resolve itself.

**Comment by brianlagunas** [comment]
Also, just to be clear, you may have to select the line of text (place your cursor on the line) you want to align before you select the alignment or nothing will happen.

**Reply by brianlagunas** [reply]
Never mind, that's not the case. I thought you could align the text lines independently, but looks like that's not how it works.
